sdks/index.html

```diff
@@ -50,6 +56,8 @@
           <a href="https://docs.aerogear.org/aerogear/latest/mobile-developer-console.html#android">Android
```

Comment:
It's not part of this change but I noticed the links for the SDK labels in the table don't work correctly. Each one navigates to the top Section of Mobile Developer Console Page.

Reply:
I don't really know where should they link to TBH, I think I will leave it for the moment.